samples/packages/asset-import/src/main/resources/package.pkl

```diff
@@ -251,6 +251,106 @@ uiConfig {
         }
       }
     }
+    ["Data Products"] {
+          description = "Data Products to import"
+          inputs {
+            ["data_products_import_type"] = new Radio {
+              title = "Import data products"
+              required = true
+              helpText = "Select how you want to provide the file containing domains and data products to be imported."
+              possibleValues {
+                ["UPLOAD"] = "Direct upload"
+                ["S3"] = "S3 object"
+              }
+              default = "UPLOAD"
+            }
+            ["data_products_file"] = new FileUploader {
+              title = "Data Products file"
+              fileTypes { "text/csv" }
+              required = false
+              helpText = "Select the file containing domains and data products to import, produced by one of the Asset Export packages."
+              placeholderText = "Select data products CSV file"
+            }
+            ["data_products_s3_region"] = new TextInput {
+              title = "S3 region"
+              required = false
+              helpText = "Enter the S3 region from which to retrieve the S3 object. If empty, will use the region of Atlan's own back-end storage."
+              placeholderText = "ap-south-1"
+              width = 4
+            }
+            ["data_products_s3_bucket"] = new TextInput {
+              title = "S3 bucket"
+              required = false
+              helpText = "Enter the S3 bucket from which to retrieve the S3 object. IF empty, will use the bucket of Atlan's own back-end storage."
```

Comment:
`IF` -> `If` (maybe a typo elsewhere in here, too?)

Reply:
Fixed